src/repositories/submission.ts

```diff
@@ -0,0 +1,63 @@
+import { SubmissionRepository, DtoSubmission, SubmissionId, Submission } from '../types/submission';
+import * as Knex from 'knex';
+import { InfraLogger as logger } from '../logger';
+import { SubmissionMapper } from '../entities/submission';
+
+export class KnexSubmissionRepository implements SubmissionRepository {
+    private readonly TABLE_NAME = 'submission';
+    // private readonly logger = new Logger(KnexSubmissionRepository.name);
```

Comment:
this need going back in?

Reply:
The logger is now the same as https://github.com/libero/continuum-adaptor/blob/master/src/logger.ts

Reply:
That being said, schema innit logic should probably be remove. .. https://github.com/libero/continuum-adaptor/blob/master/src/logger.ts

It should be part of a migration... but this PR is already large enough....

Reply:
ok - we should remove this line then?